setup.py

```diff
@@ -24,5 +24,5 @@
     author_email='oscm@edx.org',
     license='Apache',
     packages=find_packages(exclude=['*.tests']),
-    install_requires=['slumber', 'PyJWT'],
+    install_requires=['slumber >= 0.7.1', 'PyJWT >= 1.1.0'],
```

Comment:
I understand not wanting libraries to slip out from underneath us too much, but since slumber and PyJWT presumably use semver, might we change the limit to `slumber>=0.7.1,<1.0`, `PyJWT>=1.1.0,<2.0`? With my fixup commit, where we install the latest versions in CI, we should catch if a library upgrade causes problems.